domain/consensus/model/externalapi/blockEqualClone_test.go

```diff
@@ -0,0 +1,334 @@
+package externalapi
+
+import (
+	"reflect"
+	"testing"
+)
+
+type blockToCompare struct {
+	block          *DomainBlock
+	expectedResult bool
+}
+
+type TestBlockStruct struct {
+	baseBlock         *DomainBlock
+	blocksToCompareTo []blockToCompare
+}
+
+func InitTestBaseTransactions() []*DomainTransaction {
```

Comment:
Rename to `initTestBaseTransactions` so it won't be public (also applies to all the other init functions)